demo/pdfpc-video-example/Makefile

```diff
@@ -0,0 +1,20 @@
+# Suffixes
+.SUFFIXES : .tex .pdf
+
+# Executables
+PDFLATEX = pdflatex
+
+# Rules
+.tex.pdf :
+	$(PDFLATEX) $< && $(PDFLATEX) $<
+
+# Config
+DEMO  = video-example
+
+PDFDOCS = $(DEMO).pdf
+
+# Targets
+all : $(PDFDOCS)
+
+clean :
+	$(RM) $(PDFDOCS) *.aux *.log *.toc *.nav *.out *.snm *.vrb
```

Comment:
$(RM) variable is not defined. So a `make clean` raises an error for me.

Reply:
<https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html>

Hmm, I didn't realize it was GNU-specific...